GitHub Actions fails on lintr due to long lines in tests

### What happened?

Lines should not be more than 120 characters. This was violated in tests provided by #281 
https://github.com/insightsengineering/teal.slice/actions/runs/5112707542/jobs/9191072696

```{R}
2023-05-29 14:11:11 [INFO]   File:[/github/workspace/tests/testthat/test-ChoicesFilterState.R]
2023-05-29 14:11:12 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:12 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-ChoicesFilterState.R,line=314,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-ChoicesFilterState.R,line=315,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
[202](https://github.com/insightsengineering/teal.slice/actions/runs/5112707542/jobs/9191072696#step:5:203)3-05-29 14:11:12 [INFO]   ---------------------------
2023-05-29 14:11:12 [INFO]   File:[/github/workspace/tests/testthat/test-DFFilterStates.R]
2023-05-29 14:11:12 [INFO]    - File:[test-DFFilterStates.R] was linted with [R] successfully
2023-05-29 14:11:12 [INFO]   ---------------------------
2023-05-29 14:11:12 [INFO]   File:[/github/workspace/tests/testthat/test-DateFilterState.R]
2023-05-29 14:11:13 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:13 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-DateFilterState.R,line=232,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-DateFilterState.R,line=233,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:13 [INFO]   ---------------------------
2023-05-29 14:11:13 [INFO]   File:[/github/workspace/tests/testthat/test-DatetimeFilterState.R]
2023-05-29 14:11:14 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:14 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-DatetimeFilterState.R,line=247,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-DatetimeFilterState.R,line=248,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:14 [INFO]   ---------------------------
2023-05-29 14:11:14 [INFO]   File:[/github/workspace/tests/testthat/test-DefaultFilteredDataset.R]
2023-05-29 14:11:14 [INFO]    - File:[test-DefaultFilteredDataset.R] was linted with [R] successfully
2023-05-29 14:11:14 [INFO]   ---------------------------
2023-05-29 14:11:14 [INFO]   File:[/github/workspace/tests/testthat/test-EmptyFilterState.R]
2023-05-29 14:11:15 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:15 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-EmptyFilterState.R,line=25,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-EmptyFilterState.R,line=26,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:15 [INFO]   ---------------------------
2023-05-29 14:11:15 [INFO]   File:[/github/workspace/tests/testthat/test-FilterState-utils.R]
2023-05-29 14:11:16 [INFO]    - File:[test-FilterState-utils.R] was linted with [R] successfully
2023-05-29 14:11:16 [INFO]   ---------------------------
2023-05-29 14:11:16 [INFO]   File:[/github/workspace/tests/testthat/test-FilterState.R]
2023-05-29 14:11:16 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:16 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-FilterState.R,line=329,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-FilterState.R,line=330,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:16 [INFO]   ---------------------------
2023-05-29 14:11:16 [INFO]   File:[/github/workspace/tests/testthat/test-FilterStateExpr.R]
2023-05-29 14:11:17 [INFO]    - File:[test-FilterStateExpr.R] was linted with [R] successfully
2023-05-29 14:11:17 [INFO]   ---------------------------
2023-05-29 14:11:17 [INFO]   File:[/github/workspace/tests/testthat/test-FilterStates.R]
2023-05-29 14:11:18 [INFO]    - File:[test-FilterStates.R] was linted with [R] successfully
2023-05-29 14:11:18 [INFO]   ---------------------------
2023-05-29 14:11:18 [INFO]   File:[/github/workspace/tests/testthat/test-FilteredData.R]
2023-05-29 14:11:20 [INFO]    - File:[test-FilteredData.R] was linted with [R] successfully
2023-05-29 14:11:20 [INFO]   ---------------------------
2023-05-29 14:11:20 [INFO]   File:[/github/workspace/tests/testthat/test-FilteredDataset.R]
2023-05-29 14:11:20 [INFO]    - File:[test-FilteredDataset.R] was linted with [R] successfully
2023-05-29 14:11:20 [INFO]   ---------------------------
2023-05-29 14:11:20 [INFO]   File:[/github/workspace/tests/testthat/test-LogicalFilterState.R]
2023-05-29 14:11:21 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:21 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-LogicalFilterState.R,line=117,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-LogicalFilterState.R,line=118,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:21 [INFO]   ---------------------------
2023-05-29 14:11:21 [INFO]   File:[/github/workspace/tests/testthat/test-MAEFilterStates.R]
2023-05-29 14:11:22 [INFO]    - File:[test-MAEFilterStates.R] was linted with [R] successfully
2023-05-29 14:11:22 [INFO]   ---------------------------
2023-05-29 14:11:22 [INFO]   File:[/github/workspace/tests/testthat/test-MAEFilteredDataset.R]
2023-05-29 14:11:23 [INFO]    - File:[test-MAEFilteredDataset.R] was linted with [R] successfully
2023-05-29 14:11:23 [INFO]   ---------------------------
2023-05-29 14:11:23 [INFO]   File:[/github/workspace/tests/testthat/test-MatrixFilterStates.R]
2023-05-29 14:11:23 [INFO]    - File:[test-MatrixFilterStates.R] was linted with [R] successfully
2023-05-29 14:11:23 [INFO]   ---------------------------
2023-05-29 14:11:23 [INFO]   File:[/github/workspace/tests/testthat/test-RangeFilterState.R]
2023-05-29 14:11:24 [ERROR]   Found errors in [R] linter!
2023-05-29 14:11:24 [ERROR]   Error code: 31. Command output:
------
Warning: file=/github/workspace/tests/testthat/test-RangeFilterState.R,line=269,col=121,[line_length_linter] Lines should not be more than 120 characters.
Warning: file=/github/workspace/tests/testthat/test-RangeFilterState.R,line=270,col=121,[line_length_linter] Lines should not be more than 120 characters.
------
2023-05-29 14:11:24 [INFO]   ---------------------------
2023-05-29 14:11:24 [INFO]   File:[/github/workspace/tests/testthat/test-SEFilterStates.R]
2023-05-29 14:11:25 [INFO]    - File:[test-SEFilterStates.R] was linted with [R] successfully
2023-05-29 14:11:25 [INFO]   ---------------------------
2023-05-29 14:11:25 [INFO]   File:[/github/workspace/tests/testthat/test-calls_combine_by.R]
2023-05-29 14:11:25 [INFO]    - File:[test-calls_combine_by.R] was linted with [R] successfully
2023-05-29 14:11:25 [INFO]   ---------------------------
2023-05-29 14:11:25 [INFO]   File:[/github/workspace/tests/testthat/test-count_labels.R]
2023-05-29 14:11:26 [INFO]    - File:[test-count_labels.R] was linted with [R] successfully
2023-05-29 14:11:26 [INFO]   ---------------------------
2023-05-29 14:11:26 [INFO]   File:[/github/workspace/tests/testthat/test-filter_panel_api.R]
2023-05-29 14:11:27 [INFO]    - File:[test-filter_panel_api.R] was linted with [R] successfully
2023-05-29 14:11:27 [INFO]   ---------------------------
2023-05-29 14:11:27 [INFO]   File:[/github/workspace/tests/testthat/test-init_filter_state.R]
2023-05-29 14:11:27 [INFO]    - File:[test-init_filter_state.R] was linted with [R] successfully
2023-05-29 14:11:27 [INFO]   ---------------------------
```

### sessionInfo()

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

--- a/tests/testthat/test-ChoicesFilterState.R
+++ b/tests/testthat/test-ChoicesFilterState.R
@@ -311,9 +311,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- ChoicesFilterState$new(7, dataname = "data", varname = "variable")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns properly formatted string representation", {

--- a/tests/testthat/test-DateFilterState.R
+++ b/tests/testthat/test-DateFilterState.R
@@ -229,9 +229,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- DateFilterState$new(dates, dataname = "data", varname = "variable")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns a properly formatted string representation", {

--- a/tests/testthat/test-DatetimeFilterState.R
+++ b/tests/testthat/test-DatetimeFilterState.R
@@ -244,9 +244,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- DatetimeFilterState$new(posixct, dataname = "data", varname = "variable")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns a properly formatted string representation", {

--- a/tests/testthat/test-EmptyFilterState.R
+++ b/tests/testthat/test-EmptyFilterState.R
@@ -22,9 +22,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- EmptyFilterState$new(NA, dataname = "data", varname = "variable", keep_na = FALSE, disabled = TRUE)
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns a properly formatted string representation", {

--- a/tests/testthat/test-FilterState.R
+++ b/tests/testthat/test-FilterState.R
@@ -326,9 +326,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- FilterState$new(7, dataname = "data", varname = "7")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 

--- a/tests/testthat/test-LogicalFilterState.R
+++ b/tests/testthat/test-LogicalFilterState.R
@@ -114,9 +114,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- LogicalFilterState$new(logs, dataname = "data", varname = "variable")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns a properly formatted string representation", {

--- a/tests/testthat/test-RangeFilterState.R
+++ b/tests/testthat/test-RangeFilterState.R
@@ -266,9 +266,18 @@ testthat::test_that("format accepts logical show_all", {
   filter_state <- RangeFilterState$new(nums, dataname = "data", varname = "variable")
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = TRUE)))
   testthat::expect_no_error(shiny::isolate(filter_state$format(show_all = FALSE)))
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 1)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = 0)), "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'")
-  testthat::expect_error(shiny::isolate(filter_state$format(show_all = "TRUE")), "Assertion on 'show_all' failed")
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 1)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = 0)),
+    "Assertion on 'show_all' failed: Must be of type 'logical flag', not 'double'"
+  )
+  testthat::expect_error(
+    shiny::isolate(filter_state$format(show_all = "TRUE")),
+    "Assertion on 'show_all' failed"
+  )
 })
 
 testthat::test_that("format returns a properly formatted string representation", {

